src/bsp/src/stm32/hal/src/hal.c

```diff
@@ -18,7 +18,7 @@ void Hal_initMcu() {
 
     /* Reset of all peripherals, Initializes the Flash interface and the Systick. */
     HAL_Init();
-    MX_RNG_Init();
+    // MX_RNG_Init();
```

Comment:
Uncomment? Or remove if initialized somewhere else

Reply:
Ah, wait missing a TODO. But it's because the init fails